app/views/filters/index.html.erb

```diff
@@ -1,3 +1,48 @@
 <%= turbo_frame_tag 'frame_filters' do %>
-  <p>You have a total of <%= keywords_count %> keywords.</p>
+  <p>
+    <%= t('filters.total_of_keywords', keywords_count: keywords_count) %>
+  </p>
+  <%= form_with url: keywords_path, method: :get, class: 'form-filters', data: { turbo: true, turbo_frame: 'frame_list_keywords' } do |form| %>
```

Comment:
```suggestion
  <%= form_with url: keywords_path, method: :get, class: 'form-filter', data: { turbo: true, turbo_frame: 'frame_list_keywords' } do |form| %>
```

Reply:
OMG >< I will add a pre-commit check with `("|')[ \w]+s [ \w]+("|')` in the view director if this continue! 
Sorry for the repetitive omit =/

Reply:
Haha working pretty well actually!
![image](https://user-images.githubusercontent.com/77609814/127439921-7bd28e99-af3b-4919-9e82-b576142c3f71.png)